pkg/memo/header.go

```diff
@@ -0,0 +1,137 @@
+package memo
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	zetamath "github.com/zeta-chain/node/pkg/math"
+)
+
+const (
+	// Identifier is the ASCII code of 'Z' (0x5A)
+	Identifier byte = 0x5A
+
+	// HeaderSize is the size of the memo header: [identifier + ctrlByte1+ ctrlByte2 + dataFlags]
+	HeaderSize = 4
+
+	// MaskVersion is the mask for the version bits(upper 4 bits)
+	MaskVersion byte = 0b11110000
+
+	// MaskEncodingFormat is the mask for the encoding format bits(lower 4 bits)
+	MaskEncodingFormat byte = 0b00001111
```

Comment:
no need to export this. This should be a concert on the internal representation, same for other `Mask*`

Reply:
makes sense

Reply:
https://github.com/zeta-chain/node/pull/2987/commits/d022be1053ebba9c11d5d3d5c51755b3705b4c03